server/server.go

```diff
@@ -170,7 +171,6 @@ func main() {
 
 		r.Use(httpcontext.Middleware())
 		r.Use(sessions.Middleware(cookieStore))
-		r.Use(monitoring.Middleware())
```

Comment:
the middleware is already called here https://github.com/onflow/flow-playground-api/pull/347/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R113